src/appkit.cs

```diff
@@ -13868,7 +14173,7 @@ interface INSSplitViewDelegate {}
 
 	[Mac (10,10)]
 	[BaseType (typeof (NSViewController))]
-	interface NSSplitViewController : NSSplitViewDelegate {
+	interface NSSplitViewController : NSSplitViewDelegate, INSUserInterfaceValidations {
```

Comment:
You want to use `NSUserInterfaceValidations` instead or generator will pick the wrong interface.